using_images/s2i_images/nodejs.adoc

```diff
@@ -68,12 +68,25 @@ image stream definitions] for all the provided {product-title} images.
 The Node.js image supports a number of environment variables, which can be set
 to control the configuration and behavior of the Node.js runtime.
 
-To set these environment variables, you can place them into
+To set these environment variables as part of your image, you can place them into
 xref:../../dev_guide/builds.adoc#environment-files[a *_.s2i/environment_* file]
 inside your source code repository, or define them in
 xref:../../dev_guide/builds.adoc#buildconfig-environment[the environment
 section] of the build configuration's `*sourceStrategy*` definition.
 
+You can also set environment variables to be used with an existing image when
```

Comment:
We are now using xrefs instead of links, so the new format would be:
xref:../../dev_guide/new_app.adoc#specifying-environment-variables[creating new applications]
xref:../../dev_guide/environment_variables.adoc#set-environment-variables[updating environment variables for existing objects]